jaeger-core/src/test/java/com/uber/jaeger/SpanTest.java

```diff
@@ -307,15 +313,18 @@ public void testBaggageOneReference() {
   public void testBaggageMultipleReferences() {
     io.opentracing.Span parent1 = tracer.buildSpan("foo").start();
     parent1.setBaggageItem("foo", "bar");
+    this.assertBaggageLogs(parent1, "foo", "bar");
     io.opentracing.Span parent2 = tracer.buildSpan("foo").start();
     parent2.setBaggageItem("foo2", "bar");
+    this.assertBaggageLogs(parent2, "foo2", "bar");
```

Comment:
iphone's autocorrect. I edited the question. When both parent and child spans set the same baggage key, we don't record if the child overrides the parent or not.